pkgs/development/compilers/pakcs/default.nix

```diff
@@ -6,9 +6,11 @@
 let
   name = "pakcs-2.0.2";
 
+  # Don't switch to development release without a reason, because its
+  # source updates without version bump. Prefer current release instead.
```

Comment:
Apparently it was also `2.0.2` before so what did change here?

Reply:
@Mic92 there were series of betas (as seen in commit log) - 2.0.2-b7, then 2.0.2-b9. And I was forced to use betas, because 2.0.1 has fundamental issue (records support is broken) and everything lower doesn't build anymore.

Reply:
@Mic92 sorry for not updating `name` approprietary before.